### PR TITLE
Add a licence status banner

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: src/licence-status-banner/index.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    const licence = this.props.licence;
+    const licenceType = this.props.licenceType;
+    const dateFormat = this.props.dateFormat;
+
+    if (licence.status === 'active') {
+      return null;
+    }
+
+    return (
+      <div className={classnames('licence-status-banner', licence.status, this.isOpen() ? 'open' : '')}>
```
Comment (joefitter) [inline src/licence-status-banner/index.jsx]: `classnames` can take literally anything you throw at it - you could write this as:
```js
classnames('licence-status-banner', licence.status, {  open: this.isOpen() })
```

Comment (wheelsandcogs) [reply src/licence-status-banner/index.jsx]: cool, didn't know that, ta.

Comment (joefitter) [reply src/licence-status-banner/index.jsx]: SHIP IT

**Pull request merged**